devito/passes/iet/parpragma.py

```diff
@@ -527,7 +527,9 @@ def function(self):
     def imask(self):
         return self._imask
 
-    @cached_property
+    # TODO: cached_property here will break our pickling tests for reasons that
```

Comment:
out of curiosity which test?

Reply:
PRO tests landing with the `fix-udx2-MPI` branch in PRO. For future reference: they are `test_layered_funcs.py::TestMisc::testpickling` and `test_composite_funcs.py::TestMisc::testpickling`